modules/mobile/spec/request/v1/immunizations_request_spec.rb

```diff
@@ -0,0 +1,493 @@
+# frozen_string_literal: true
+
```

Comment:
The existing specs are not namespaced under v0. Do we want to add that namespace or come up with a different scheme?